src/Adapter/MSTestAdapter.PlatformServices/Utilities/AppDomainUtilities.cs

```diff
@@ -248,7 +248,6 @@ private static void EnsureRelevantStaticStateIsRestored(AppDomain appDomain)
         Type staticStateHelperType = typeof(StaticStateHelper);
         var staticStateHelper = appDomain.CreateInstanceFromAndUnwrap(staticStateHelperType.Assembly.Location, staticStateHelperType.FullName) as StaticStateHelper;
         staticStateHelper?.SetUICulture(CultureInfo.DefaultThreadCurrentUICulture);
-        staticStateHelper?.SetTestIdGenerationStrategy((int)DataRowAttribute.TestIdGenerationStrategy, (int)DynamicDataAttribute.TestIdGenerationStrategy);
```

Comment:
No longer needed now that `TestAssemblySettings` is getting called from the same AppDomain as the `UnitTestRunner`.